src/NzbDrone.Core/Download/Clients/Tribler/Models/TriblerSettingsApi.cs

```diff
@@ -0,0 +1,321 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NzbDrone.Core.Indexers.Tribler
+{
+    public class TriblerSettingsResponse
+    {
+        [JsonProperty("settings", NullValueHandling = NullValueHandling.Ignore)]
+        public Settings Settings { get; set; }
+    }
+
+    public class Settings
+    {
+        [JsonProperty("api", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Does this need `NullValueHandling = NullValueHandling.Ignore` everywhere?

Reply:
no, it's not nessesary, i just completely missed that file.